source/hpack.c

```diff
@@ -255,18 +251,18 @@ struct aws_hpack_context {
 
     struct {
         struct aws_http_header *buffer;
-        uint32_t max_elements;
-        uint32_t num_elements;
-        uint32_t index_0;
+        size_t max_elements;
```

Comment:
Is there a chance we're going to have gigs of entries in here on 64-bit machines?

Reply:
It's possible! Who knows what kind of hardware people will run this on.